Formula/par.rb

```diff
@@ -15,7 +15,7 @@ class Par < Formula
 
   conflicts_with "rancid", because: "both install `par` binaries"
 
-  # A patch by Jérôme Pouiller that adds support for multibyte
+  # A patch by Jerome Pouiller that adds support for multibyte
```

Comment:
Very tragic, given the contents of the comment and the patch provided by this fine man.

I think we could bend the rules here and be more welcoming to people that use those “funny characters” in their names or their mother tongues. And I'm not aware of any user reports where this has caused any issues, so I don't see any harm in it.